src/components/organisms/HelpCenter/HelpCenter.tsx

```diff
@@ -25,18 +27,32 @@ export const HelpCenter: React.FC = () => {
     [startLoading]
   );
 
+  const handleBackClicked = () => {
+    setUrl("");
+  };
+
   return (
     <>
       {url ? (
         <>
           {isLoading && <LoadingPage />}
-          <iframe
-            className={"HelpCenter__iframe"}
-            onLoad={stopLoading}
-            frameBorder="0"
-            src={url}
-            title="Help Center"
-          />
+          <div className={"HelpCenter__iframeContainer"}>
+            {!isLoading && (
+              <Button
+                customClass={"HelpCenter__backButton"}
```

Comment:
```suggestion
                customClass="HelpCenter__backButton"
```

Reply:
@yurii-lubynets fixed in 5fe10535b862a45805d69a34bcfcbfe5e1666ef1